ocaml/xapi/xapi_mgmt_iface.ml

```diff
@@ -187,21 +195,19 @@ let on_dom0_networking_change ~__context =
         debug "Changing Host.address in database to: %s" ip;
         Db.Host.set_address ~__context ~self:localhost ~value:ip;
         debug "Refreshing console URIs";
+        update_getty ();
         Dbsync_master.refresh_console_urls ~__context
       end
     | None ->
       if Db.Host.get_address ~__context ~self:localhost <> "" then begin
         debug "Changing Host.address in database to: '' (host has no management IP address)";
+        update_getty ();
         Db.Host.set_address ~__context ~self:localhost ~value:""
       end
   end;
   Helpers.update_domain_zero_name ~__context localhost new_hostname;
   (* Running update-issue service on best effort basis *)
-  try
-    ignore (Forkhelpers.execute_command_get_output !Xapi_globs.update_issue_script []);
-    ignore (Forkhelpers.execute_command_get_output !Xapi_globs.kill_process_script ["-q"; "-HUP"; "mingetty"; "agetty"])
-  with _ -> ();
-    debug "Signalling anyone waiting for the management IP address to change";
-    Mutex.execute management_ip_mutex
-      (fun () -> Condition.broadcast management_ip_cond)
+  debug "Signalling anyone waiting for the management IP address to change";
```

Comment:
Wow, looks like this was part of the exception handler before, but it should not have been!!